mailer.rst

```diff
@@ -0,0 +1,649 @@
+Sending Emails with Mailer
+==========================
+
+.. versionadded:: 4.3
+    The Mailer component was added in Symfony 4.3. The previous solution - Swift Mailer -
+    is still valid: :doc:`Swift Mailer</email>`.
+
+Installation
+------------
+
+.. caution::
+
+    The Mailer component is experimental in Symfony 4.3: some backwards compatibility
+    breaks could occur before 4.4.
+
+Symfony's Mailer & :doc:`Mime </components/mime>` components form a *powerful* system
+for creating and sending emails - complete with support for multipart messages, Twig
+integration, CSS inlining, file attachments and a lot more. Get them installed with:
+
+.. code-block:: terminal
+
+    $ composer require symfony/mailer
+
+Transport Setup
+---------------
+
+Emails are delivered via a "transport". And without installing anything else, you
+delivery emails over ``smtp`` by configuring your ``.env`` file:
```

Comment:
sentence seems wrong when reading, is a word missing?

Reply:
good catch!